### PR TITLE
WDAC/Recommended block rules: Add notes and link

### LINKED ISSUE #3642: Ambiguity
"Beginning with the March 2019 quality update, each version of Windows requires blocking a specific version of the following files:"
Does this mean that (for example) on Windows 1803, I only block the files commented as &lt;! -- RS4 Windows 1803 --&gt;? In that case, what do I do if I need a policy that works on both Windows 1803 and 1809? The implication is that I need two policies. But why not just block all versions of those files?
Please explain more clearly what the block list should contain for these three files.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9d71055e-2b14-227b-0107-c8f150194d5b
* Version Independent ID: 0eab18a0-7e8f-05eb-b47f-e6913682d8e8
* Content: [Microsoft recommended block rules (Windows 10)](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-block-rules)
* Content Source: [windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-block-rules.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-block-rules.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @jsuther1974
* Microsoft Alias: **justinha**

### PR DESCRIPTION
**Pull request by illfated**
**Description:**

As agreed on in issue ticket #3642 (**Ambiguity**), this PR adds a note about creating two diff policies or merging them in a broad policy,
as well as adding a link to how to merge WDAC policies. 

As recommended by @Justinha , Windows Server 2019 is also added to the "Applies to" list at the top of this document.

Also, thanks to @Air-Git for requesting this clarification.

**Proposed changes:**
- add note about creating two diff policies or merging them in a broad policy
- add a link to how to merge WDAC policies
- add Windows Server 2019 to the "Applies to" list
- Cosmetic: add single compatibility MarkDown quote spacing to the existing Note.

**issue ticket closure or reference:**

Closes #3642

### CODE REVIEW

**Comment by illfated** [comment]
Rich Text Diff view, for easy overview of the changes:
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/5112/files?short_path=540ffe4#diff-540ffe49bd32e95e8b2a5da5ba1a565e

Suggestions and recommendations are welcome, especially for correct wording in the link section heading at the end.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-block-rules.md [hunk 1]
```diff
@@ -1511,3 +1512,10 @@ Pick the correct version of each .dll for the Windows release you plan to suppor
   </SiPolicy>
 ```
 <br />
+
+> [!Note]
+> To create a policy that works on both version 1803 and 1809, you can create two diff policies, or merge them into one broader policy.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-block-rules.md]:
```suggestion
> To create a policy that works on both Windows 10, version 1803 and version 1809, you can create two different policies, or merge them into one broader policy.
```

File: windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-block-rules.md [hunk 2]
```diff
@@ -1511,3 +1512,10 @@ Pick the correct version of each .dll for the Windows release you plan to suppor
...
+
+## Further reading (How-To)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-block-rules.md]:
```suggestion
## More information
```

**Comment by illfated** [comment]
Updated content, simplified in Rich Text Diff view:
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/5112/files?short_path=540ffe4#diff-540ffe49bd32e95e8b2a5da5ba1a565e

**Pull request merged**